gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/step/Mutating.java

```diff
@@ -28,7 +28,7 @@
  * @author Stephen Mallette (http://stephen.genoprime.com)
  * @author Matt Frantz (http://github.com/mhfrantz)
  */
-public interface Mutating<E extends Event> extends Configuring {
+public interface Mutating<E extends Event> {
```

Comment:
This change should be in the CHANGELOG and Upgrade Docs (providers section) as it might break strategies.